common/src/IO/GameConfigParser.cpp

```diff
@@ -218,13 +229,29 @@ namespace TrenchBroom {
             std::vector<Model::FlagConfig> flags;
             for (size_t i = 0; i < value.length(); ++i) {
                 const EL::Value& entry = value[i];
-
-                expectStructure(entry, "[ {'name': 'String'}, {'description': 'String'} ]");
-
-                const std::string& name = entry["name"].stringValue();
-                const std::string& description = entry["description"].stringValue();
-
-                flags.push_back(Model::FlagConfig(name, description));
+                bool unused;
+                if (m_version == 3) {
+                    unused = false;
+                    expectStructure(entry,
+                                    "["
+                                    "{'name': 'String'},"
+                                    "{'description': 'String'}"
+                                    "]");
+                } else {
+                    unused = entry["unused"].booleanValue();
+                    if (!unused) {
+                        expectStructure(entry,
```

Comment:
Why not always expect this structure?

Reply:
If the flag is unused there's no need to specify any other properties like "name".

Reply:
I could add some checking in the unused case just to enforce that no other weird properties sneak in, while still recognizing that a name is not required. Something like:
```c++
if (unused) {
    expectStructure(value,
                    "["
                    "{},"
                    "{'name': 'String', 'description': 'String', 'unused': 'Boolean'}"
                    "]");
} else {
    expectStructure(value,
                    "["
                    "{'name': 'String'},"
                    "{'description': 'String', 'unused': 'Boolean'}"
                    "]");
}
```